1.data-exploration/scripts/nbconverted/1.model_data_visualization.py

```diff
@@ -124,38 +123,23 @@
 # In[8]:
 
 
-age_categories = []
 age_distribution = []
 
 # Loop through each age entry to clean it
-for age_entry in model_df.age.tolist():
-    try:
-        # If the age is an integer, apply appropriate category
-        if int(age_entry) >= adult_threshold:
-            age_categories.append("Adult")
-        else:
-            age_categories.append("Pediatric")
-
-        # If the age is an integer, apply appropriate continuous measure
+for age_entry in model_df.Age.tolist():
+    try: 
+        # If the age is an integer or float, apply appropriate continuous measure
         age_distribution.append(int(age_entry))
-
-    except ValueError:
+    except:
         # If conversion fails, categorize appropriately
-        if pd.notnull(age_entry):
-            age_categories.append(age_entry)
-        else:
-            age_categories.append("Missing")
-
         age_distribution.append(np.nan)
 
 
 # In[9]:
 
 
 # Add columns age_categories & age_distribution
-model_df = model_df.assign(
-    age_categories=age_categories, age_distribution=age_distribution
-)
+model_df = model_df.assign(age_distribution=age_distribution)
```

Comment:
Are you using `age_distribution` anywhere anymore? If not, please remove the vestigial code

Reply:
I was using it to create the age distribution graph, but that can be substituted with the included `Age` column. Additionally, I realized cleaning the `Age` variable was no longer necessary with the new dataset.